distribution/metadata/v1_id_service_test.go

```diff
@@ -22,6 +22,11 @@ func TestV1IDService(t *testing.T) {
 	}
 	v1IDService := NewV1IDService(metadataStore)
 
+	ns := v1IDService.namespace()
+	if ns != "v1id" {
+		t.Fatal("namespace return must v1id")
+	}
```

Comment:
This failure message is missing one of the two most important pieces of information, the actual value.

Please use 

```go
require.Equal(t, "v1id", ns)
```